astropy/table/table.py

```diff
@@ -896,24 +900,30 @@ def show_in_notebook(self, tableid=None, css=None, display_length=50,
                                             np.random.randint(1, 1e6))
 
         jsv = JSViewer(display_length=display_length)
-        html = self._base_repr_(html=True, max_width=-1, tableid=tableid,
-                                max_lines=-1, show_dtype=False,
-                                tableclass=table_class)
+
+        has_row_index = 'row_index' in self.colnames
```

Comment:
@saimn - ah, good catch, that's a bug - it definitely shouldn't be added when it's already there